README.md

```diff
@@ -4,7 +4,10 @@
 **We are in a really early Alpha release. You should be ready for hard adventures. 
 If you have updated to version 0.0.2 or greater - please re-download all pre-trained models**
 
-DeepPavlov is an open-source conversational AI library built on [TensorFlow](https://www.tensorflow.org/) and [Keras](https://keras.io/). It is designed for
+### *We are in a really early Alpha release. You should be ready for hard adventures.*
+### *If you have updated to version 0.0.2 or greater - please re-download all pre-trained models*
+
+DeepPavlov is an open-source conversational AI library built on TensorFlow and Keras. It is designed for
```

Comment:
alpha warning is there twice, TensorFlow and Keras links are lost

Reply:
I will fix it